gapic/src/main/com/google/gapid/perfetto/models/FrameEvents.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.perfetto.models;
+
+import static com.google.gapid.util.MoreFutures.transform;
+
+import com.google.common.collect.Iterables;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.gapid.models.Perfetto;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Data about the tracks for the Frame Events.
+ */
+public class FrameEvents {
```

Comment:
I tried doing this part but since the nested classes of GpuInfo are static, this is causing a lot of issues for the constructor. If you create two different GpuInfo objects, one for gpu queue and the other for frameEvents, and have two separate lists for them, the one that is created last has its members retained. For example, if you do frameEvents = null while constructing gpu queue and gpu_queue = null while constructing frameEvents, this is causing gpu_queue to be null even with the gpu object, thereby failing at runtime. 

I agree with renaming the FrameTrack. I'll do it in the next commit.